portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -135,3 +126,16 @@ function insertSearch() {
 
   document.getElementById('mainNav').appendChild(searchBar);
 }
+
+async function addUserComments() {
+  const response = await fetch('/map');
```

Comment:
Why the endpoint of "user comments" is named as "map"?

Reply:
I store the user-submitted data in the map endpoint and serve the results through the same servlet. I only named it 'map' to denote what the servlet is being used for.

Reply:
So what does map endpoint do?

Reply:
Stores the user-submitted info into the database and serves it back to the javascript when requested

Reply:
I have changed the name of the endpoint.